playbooks/roles/hypershift/defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+# hypershift default variables
+hypershift_install: false
+hypershift_destroy_cluster: false
+mgmt_cluster_kubeconfig: "{{ ansible_env.HOME }}/.kube/config"
+hosted_cluster_name: "rdr-hypershift-cluster"
+hypershift_dir: "/root/hypershift"
+hypershift_git_repo: "https://github.com/openshift/hypershift"
+hypershift_git_branch: "main"
+ibmcloud_resource_group: ""
+powervs_region: ""
+powervs_zone: ""
+vpc_region: ""
+base_domain: ""
```

Comment:
As these are required variables there is no use defining the empty default values which may lead to failures.